### ISSUE TITLE
/dev/shm read/write issue with Docker

### ISSUE THREAD

**Issue by rvernica** [travis-ci/travis-ci#8917]
In my `.travis.yml` I start a Docker container like this:
```
before_install:
  - docker pull rvernica/scidb:${SCIDB_VER}-ext
  - pwd
  - docker run
    --name scidb
    --detach
    --volume /dev/shm
    --tmpfs /dev/shm
    --volume `pwd`:/shim
    --publish 8080:8080
    rvernica/scidb:${SCIDB_VER}-ext
```
One of the test scripts uses `/dev/shm` for read/write. This worked fine before the December 12, 2017 update. Now it breaks. Here is an [example](https://travis-ci.org/Paradigm4/shim/builds/304979732#L828). This build worked fine before December 12. I triggered the rebuild now and it fails.

I configured Travis to use the previous build as instructed [here](https://blog.travis-ci.com/2017-12-12-new-trusty-images-q4-launch) and the build succeeds.

This might be a Docker bug.

**Comment by BanzaiMan** [comment]
Could you confirm that the new version of Docker fails when using the old image? https://docs.travis-ci.com/user/docker/#Installing-a-newer-Docker-version

**Comment by rvernica** [comment]
Yes, I can [confirm](https://travis-ci.org/Paradigm4/shim/builds/316770872#L458) that. I got Docker upgraded in the old image to `17.11.0` (a bit newer that in the new image) and the build fails.

**Comment by BanzaiMan** [comment]
Thanks for testing. I tend to think this is a Docker issue. I am not sure if downgrading Docker is an option, but it is worth trying.
